src/Pages/Profile/MyAccount.tsx

```diff
@@ -117,29 +117,28 @@ const MyAccount: React.FC = () => {
                 return false; // Prevent default upload behavior
               }}
             >
-              {' '}
               {imageUrl ? (
-                <img src={imageUrl} alt="avatar" style={{ width: '100%' }} />
+                <img src={imageUrl} alt="avatar" className="profile-image" />
```

Comment:
For some reason I can upload a profile picture, but it doesn't remain. And it didn't use the profile picture that I was using before. Maybe just an issue of this test-deployment, but haven't noticed that in the past.